src/libtriton/includes/triton/hash.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef TRITON_HASH_HPP
+#define TRITON_HASH_HPP
+
+#include <triton/tritonTypes.hpp>
+
+namespace triton
+{
+
+template<typename T>
+class IdentityHash
+{
+public:
+  // KEy is unique and are be returned as is
+  T operator()(const T& key) const { return key; }

```

Comment:
I think we can delete this file and define the `IdentityHash` into `tritonTypes.hpp`. Also, we may add to the comment something saying that this is used by `robin_map`.